Custom datasource adapter (#36)
Knotx/knotx-example-project: README.md

    - [http://localhost:8092/content/simple.html](http://localhost:8092/content/simple.html)
    - [http://localhost:8092/content/login/step1.html](http://localhost:8092/content/login/step1.html)
    - [http://localhost:8092/content/multiple-forms.html](http://localhost:8092/content/multiple-forms.html)

>     - [http://localhost:8092/content/db-books.html](http://localhost:8092/content/db-books.html) _Note_: to have it working you need to run local DB. Details can be found [Readme](https://github.com/Knotx/knotx-example-project/tree/master/knotx-data-bridge/acme-db-datasource-adapter) 

Reviewer: ```suggestion
    - [http://localhost:8092/content/db-books.html](http://localhost:8092/content/db-books.html) _Note_: to have it working you need to run your local HSQLDB instance. More details can be found [here](https://github.com/Knotx/knotx-example-project/tree/master/knotx-data-bridge/acme-db-datasource-adapter). 
```
